pkg/services/rendering/rendering.go

```diff
@@ -110,6 +113,23 @@ func (rs *RenderingService) RenderErrorImage(err error) (*RenderResult, error) {
 }
 
 func (rs *RenderingService) Render(ctx context.Context, opts Opts) (*RenderResult, error) {
+	startTime := time.Now()
+	result, err := rs.render(ctx, opts)
+	elapsedTime := time.Since(startTime).Milliseconds()
+	if status.Code(err) == codes.DeadlineExceeded {
+		metrics.MRenderingRequestTotal.WithLabelValues("cancelled").Inc()
+		metrics.MRenderingSummary.WithLabelValues("cancelled").Observe(float64(elapsedTime))
```

Comment:
I'm not sure should we record summary for cancelled requests. Duration for these requests is always the same as timeout. But in the other hand, timeout can be changed and in this case metric may make sense.

Reply:
Timeouts could be different from UI and Grafana alerting. But you're probably right we shouldn't include summary of cancelled requests.

Reply:
:wave: I think we should include all requests in the summary. Regardless if it times out or not. It that's that more consistent. 

If someone claims that the render service is slow and you go look at the numbers you say its not true. If you have an SLO for how fast you should process requests, ignoring some would be deceiving.

Reply:
Ok, that makes sense as well.